### ISSUE TITLE
node-red security (securing after firewall sure?!)

### ISSUE THREAD

**Issue by gitmacer** [Uberspace/lab#1805]
in https://lab.uberspace.de/guide_node-red/
The firewall rule is added before the password gets set.
Are you sure that is agood idea?

**Comment by EV21** [comment]
You are right, the sections `Configure web server` and `Securing` should switch place. 
I`m just a contributor like you, so feel free to make a pull request so it could be fixed soon™.
